lib/View/ListView.js

```diff
@@ -96,6 +96,9 @@ class ListView extends View {
         }
 
         this._filters = orderElement.order(filters);
+        
+        // a filter cannot be required
+        this._filters.map((filter) => filter.validation({ required: false }));
```

Comment:
can potentially change the configuration of a form if the field is passed in a view too.